app/main.py

```diff
@@ -1,3 +1,49 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | Distance) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | Distance) -> Distance:
+        return Distance(self.km * other)
+
+    def __round__(self) -> int:
+        return round(self.km)
+
+    def __truediv__(self, other: int | Distance) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: int) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: int) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: int) -> bool:
+        return self.km == other
+
+    def __le__(self, other: int) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: int) -> bool:
+        return self.km >= other
```

Comment:
what about it?

Reply:
do you mean annotation?

Reply:
yes

Reply:
nothing passes except bool! If other - give a hint, please!

Reply:
other can be float or Distance

Reply:
You should fix requested changes all over the project. `other` can be not only `int`